cards.md

```diff
@@ -112,7 +112,9 @@ fileName:cardResponse.json
 
 ## Metadata
 
-The Card object can also have associated metadata, with an `id` property that is a *non-unique index*.  When creating a card, you can set the `id` property of the metadata to a custom identifier, for example `my-brand-name-cards`. Later you can use our Cards API to retrieve a list of cards by using the metadata `ID`. You can read more about this in our API Reference for the [List Cards from Metadata ID](https://reference.fidel.uk/v1/reference/list-cards-from-metadata-id) endpoint.
+The Card object can also have associated metadata, i.e., auxiliary data to better describe or help tracking and working with cards. The metadata field requires an `id` mandatory property, which works as a *non-unique index* for retrieval purposes. We recommend that no confidential or sensitive data regarding the cardholder be stored in clear in this property.
```

Comment:
I would break this text down into shorter phrases, and make them more simple. For example (and I hope I don't make wrong assumptions, so please take it with a grain of salt from the technical perspective):
"The metadata field is an object. It must contain a property called `id`, which is a non-unique index. You can use the `id` property to retrieve .... (please add what they can retrieve and how). It's recommended not to include unencoded confidential or sensitive data regarding the cardholder.

Reply:
@krisztina-nagy based on your comments I adapted the text.